tests/test_graph.py

```diff
@@ -102,6 +103,17 @@ def test__isub__(self):
         assert g.top == 'b'
         assert g is original
 
+    def test__contains__(self):
+        g = Graph([('a', ':instance', 'alpha'),
+                   ('a', ':ARG', 'b'),
+                   ('b', ':instance', 'beta')])
+
+        assert "alpha" in g
+        assert "beta" in g
+        assert "b" not in g
```

Comment:
This one is confusing because `b` is in g, it's just not defined as such. Not sure what the best path forward is. But you could have something obviously false, like `assert "gamma" not in g`